plugin/figure/src/main/java/io/deephaven/figure/FigureWidgetTranslator.java

```diff
@@ -722,6 +724,7 @@ private SourceDescriptor makeSourceDescriptor(IndexableNumericData data, SourceT
 
             source.setColumnName(columnHandler.getColumnName());
             source.setTableId(tablePositionMap.get(columnHandler.getTableHandle()));
+            source.setPartitionedTableId(-1);
```

Comment:
Is `-1` a canonical way to express "null"?

Reply:
It is what the API expects for "unset, don't try to index into the list of tickets with this" - PB returns zero for unset values, which clearly is a valid index, so -1 is "this is explicitly unset".